docs/reference/manifest/version.md

```diff
@@ -1,23 +1,26 @@
 ---
 title: Version element in the manifest file
 description: The Version element specifies your Office Add-in version.
-ms.date: 10/09/2018
+ms.date: 02/05/2021
 localization_priority: Normal
 ---
 
 # Version element
 
-Specifies the version of your Office Add-in.
+Specifies the version of your Office Add-in. The version number can be 1, 2, 3, or 4 parts (i.e., n, n.n, n.n.n, or n.n.n.n).
 
 **Add-in type:** Content, Task pane, Mail
 
 ## Syntax
 
 ```XML
-<Version>n .n .n .n</Version>
+<Version>n(.n.n.n)</Version>
```

Comment:
I think its more standard to use square brackets [] rather than parentheses () to indicated optional things.

Reply:
Fixed. Good call.